### PR TITLE
add youtube playlists to seminars page

### PR DESCRIPTION
**Pull request by carinaluise**

### CODE REVIEW

**Review by h0rhay — COMMENTED** [review]
Are we able to keep the cards, but add the yt videos as thumbnails for the cards?
It feels like we have lost design polish for this page now?

**Comment by carinaluise** [comment]
@h0rhay added a rounded utility class on the videos and implemented a flex layout 👍

File: src/pages/ecosystem/resources/past-seminars.js [hunk 1]
```diff
@@ -12,27 +17,37 @@ export default function PastSeminars() {
...
-                </div>
-              </div>
-            </Link>
-          ))}
+        <h2 className="text-2xl font-normal font-body mb-6">Substrate Seminar Live Streams (2021-2022)</h2>
+        <div className="flex flex-wrap gap-8">
+          {oldPlaylists.map((playlist, idx) => {
+            return (
+              <iframe
+                key={idx}
```
Comment (h0rhay) [inline src/pages/ecosystem/resources/past-seminars.js:41]: FYI, not a showstopper.. but prob best to use the video IDs as the react key.. as not really the 'done' thing to use `index` from the array in prod.
The YT IDs will always be unique.. so fine to use these.

**Review by h0rhay — APPROVED** [review]
Changes look good.
1 comment re not using `index` for react keys.
The UI for these vids is now way better, really good work.

**Pull request merged**